app/server/methods/homes.js

```diff
@@ -270,5 +270,38 @@ Meteor.methods({
     }
 
     return Homes.find(selector).map(home => home._id);
+  },
+
+  assignManager({ groupId, userId }) {
+    // Get current user ID
+    const currentUserId = Meteor.userId();
+
+    // Check if current user is Admin
+    const currentUserIsAdmin = Roles.userIsInRole(currentUserId, "admin");
+    if (!currentUserIsAdmin) throw new Meteor.Error(500, 'User does not have right to perform this action')
+    if (!groupId || !userId)
+      throw new Meteor.Error(500, "Group or user not specified!");
+
+    Permissions.update({ groupId }, { $set: { isManager: false } })
```

Comment:
This line of code seems dangerous. I.e. it seems like it will update **all** permissions for the `groupId` and set `isManager` to `false`. If this is the case, it would leave the home without any managers.

Reply:
Basically, this line of code might not need to be here at all.